engine/src/main/java/org/terasology/engine/subsystem/headless/renderer/HeadlessWorldRenderer.java

```diff
@@ -300,4 +295,15 @@ public int compare(RenderableChunk o1, RenderableChunk o2) {
         }
     }
 
+	@Override
+	public RenderingConfig getRenderingConfig() {
```

Comment:
I tried doing it via injection, as you suggested, but the game keeps crashing on me when I do it, I'll keep trying to implement it with injections, but for now I've managed to use setters to make it so that the method doesn't need parameters